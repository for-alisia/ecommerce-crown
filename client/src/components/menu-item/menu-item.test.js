// @ts-nocheck

/** Libraries */
import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';

/** Components */
import { MenuItemContainer } from './menu-item.styles';

describe('Custom button component', () => {
  const wrapper = shallow(<MenuItemContainer />);

  it('expects to match a snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
