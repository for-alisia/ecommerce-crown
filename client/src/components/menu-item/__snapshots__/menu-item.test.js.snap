// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Custom button component expects to match a snapshot 1`] = `
<div
  className="bJKGbA"
/>
`;
